service/src/main/kotlin/xyz/lebkuchenfm/external/security/Pbkdf2PasswordEncoder.kt

```diff
@@ -0,0 +1,23 @@
+package xyz.lebkuchenfm.external.security
+
+import xyz.lebkuchenfm.domain.security.HashedPasswordHexEncoded
+import xyz.lebkuchenfm.domain.security.PasswordEncoder
+import javax.crypto.SecretKeyFactory
+import javax.crypto.spec.PBEKeySpec
+
+class Pbkdf2PasswordEncoder : PasswordEncoder {
+    private val factory = SecretKeyFactory.getInstance("PBKDF2WithHmacSHA512")
+    private val iterationCount = 50000
```

Comment:
it is recommended to use 210,000 iterations.
https://cheatsheetseries.owasp.org/cheatsheets/Password_Storage_Cheat_Sheet.html#pbkdf2

Reply:
We've used 50000 iterations in v2. Changing this value will break currently saved credentials - forcing everyone to reset their passwords.

Reply:
Do we want to add it to known problems?

Reply:
It already is :)